packages/gatsby-plugin-google-analytics/src/gatsby-browser.js

```diff
@@ -1,7 +1,7 @@
-exports.onRouteUpdate = function(location) {
+exports.onRouteUpdate = function({ location }) {
```

Comment:
Added destructuring, I guess I misread the [plugin docs](https://www.gatsbyjs.org/docs/plugins/) regarding the `src` folder.